analysis/src/Packages.ml

```diff
@@ -148,61 +148,6 @@ let newBsPackage ~rootPath =
              pathsForModule;
              opens;
              namespace;
-             builtInCompletionModules =
-               (if
-                  opens_from_bsc_flags
-                  |> List.find_opt (fun opn ->
-                         match opn with
-                         | ["RescriptCore"] -> true
-                         | _ -> false)
-                  |> Option.is_some
-                  || fst rescriptVersion >= 12
-                then
-                  {
-                    arrayModulePath = ["Array"];
-                    optionModulePath = ["Option"];
-                    stringModulePath = ["String"];
-                    intModulePath = ["Int"];
-                    floatModulePath = ["Float"];
-                    promiseModulePath = ["Promise"];
-                    listModulePath = ["List"];
-                    resultModulePath = ["Result"];
-                    exnModulePath = ["Exn"];
-                    regexpModulePath = ["RegExp"];
-                  }
-                else if
-                  opens_from_bsc_flags
-                  |> List.find_opt (fun opn ->
-                         match opn with
-                         | ["Belt"] -> true
-                         | _ -> false)
-                  |> Option.is_some
-                then
-                  {
-                    arrayModulePath = ["Array"];
-                    optionModulePath = ["Option"];
-                    stringModulePath = ["Js"; "String2"];
-                    intModulePath = ["Int"];
-                    floatModulePath = ["Float"];
-                    promiseModulePath = ["Js"; "Promise"];
-                    listModulePath = ["List"];
-                    resultModulePath = ["Result"];
-                    exnModulePath = ["Js"; "Exn"];
-                    regexpModulePath = ["Js"; "Re"];
-                  }
-                else
-                  {
-                    arrayModulePath = ["Js"; "Array2"];
-                    optionModulePath = ["Belt"; "Option"];
-                    stringModulePath = ["Js"; "String2"];
-                    intModulePath = ["Belt"; "Int"];
-                    floatModulePath = ["Belt"; "Float"];
-                    promiseModulePath = ["Js"; "Promise"];
-                    listModulePath = ["Belt"; "List"];
-                    resultModulePath = ["Belt"; "Result"];
-                    exnModulePath = ["Js"; "Exn"];
-                    regexpModulePath = ["Js"; "Re"];
-                  });
```

Comment:
`builtInCompletionModules` are not needed now that Core is in the compiler repo. But, we are going to make these configurable through the project config soon.